doc/index.rst

```diff
@@ -34,9 +34,18 @@ There are three different ways which you can use Cube Browser:
 Installation
 ____________
 
-Conda is great
+In the following pages you will find downloads of Jupyter notebooks containing Cube Browser operations.
+To be able to use these notebooks, you will need to configure your environment correctly.
+The most efficient and clean method to do this is to use a Conda environment.
+
+If you have never used Conda before, you will need to follow these `quick installation instructions <http://conda.pydata.org/docs/install/quick.html>`_ to get started.
+
+Once you have done this, open a terminal and type:
+
+``conda install -c scitools -c conda-forge -c scitools/label/dev cube_browser``
```

Comment:
this is not the approach we have taken with other scitools docs

for iris and for cartopy we have provided a 
```
conda install
```

instruction

further use of conda we have left to advanced conda users

i think i prefer to keep it this way, and when we decide to update our conda advice, do it consistently across scitools package docs